sentry-android-core/src/test/java/io/sentry/android/core/PerformanceAndroidEventProcessorTest.kt

```diff
@@ -107,6 +111,31 @@ class PerformanceAndroidEventProcessorTest {
         assertTrue(tr.measurements.isEmpty())
     }
 
+    @Test
+    fun `do not add slow and frozen frames if not auto transaction`() {
+        val sut = fixture.getSut()
+        var tr = getTransaction("task")
+
+        tr = sut.process(tr, null)
+
+        assertTrue(tr.measurements.isEmpty())
+    }
+
```

Comment:
I wonder if it compiles ;)

```suggestion
    @Test
    fun `do not add slow and frozen frames if not auto transaction`() {
        val sut = fixture.getSut()
        var tr = getTransaction("task")

        tr = sut.process(tr, null)

        assertTrue(tr.measurements.isEmpty())
    }

    @Test
    fun `do not add slow and frozen frames if tracing is disabled`() {
        val sut = fixture.getSut(0)
        var tr = getTransaction("task")

        tr = sut.process(tr, null)

        assertTrue(tr.measurements.isEmpty())
    }

```

Reply:
almost, sut gets a Double, so only 0 wont work :D
added the test, although the same use case was already tested by `do not add app start metric if performance is disabled` since the `if` is above both blocks, added anyway as it makes sense.